concepticondata/conceptlists.tsv

```diff
@@ -322,4 +322,5 @@ Sims-2020-253	Nathaniel A. Sims	2020		253	proto-language, areal	English	Northeas
 Josserand-1983-188	Josserand, Judy Katherine	1983		188	basic, areal	Spanish, English	Mixtec languages		Josserand1983	Josserand1983	This concept list consists of 188 cognate sets [Josserand (1983)](:bib:Josserand1983) used for subgrouping. The cognate sets cover 120 varieties (coverage differs across cognate sets) and a proto-Mixtec reconstruction for each concept. Tones are not marked.	489-678	
 Duerr-1987-110	Dürr, Michael	1987		110	basic, areal	English	Mixtec languages		Durr1987	Duerr1987	This concept list consists of 110 cognate sets across 17 varieties of Mixtec including a proto-Mixtec reconstruction for each. It was used by [Dürr (1987)](:bib:Durr1987) for tone reconstruction. It overlaps to some degree with Josserand 1983.	38-55	
 Zhang-2019-308	Zhang, Shuya and Jacques, Guillaume and Lai, Yunfan	2019		308	basic		rGyalrong and Old Chinese	https://pure.mpg.de/pubman/faces/ViewItemOverviewPage.jsp?itemId=item_3149499	Zhang2019		This list of 308 items was extracted from a study on cognates among rGyalrong languages and Old Chinese. Since the data is a collection of etymological items with very divergent meanings and elicitation glosses, we assembled the data for each language separately and added a unified layer of translation.	73-92	
+Wu-2020-150	Wu, Winston  and Nicolai, Garrett  and Yarowsky, David	2020		150	basic, ranked	English	Global	https://www.aclweb.org/anthology/2020.lrec-1.519	Wu2020	Wu2020	The given list was created as a result of the automatic creation of the core vocabulary, based on the relative coverage of each target concept across 1895 bilingual dictionaries found in the LanguageNet multiligual lexicon [(Baldwin et al., 2010)](:bib:Baldwin2010). It represents a subset of 150 most frequent glosses from the larger 3000-item vocabulary, sorted in descending order.	4212	
```

Comment:
- I would go for "The given list is a result of the automatic creation..." to avoid the doubling of was created/creation.
- remove comma after Baldwin et al. in the citation
- "It represents a subset of **the** 150 most frequent glosses..."
- Probably just some weird rendering, but it looks like there might be multiple spaces in front of the "and"s in the author name column?